psql2mysql.py

```diff
@@ -199,7 +230,8 @@ def do_prechecks(config):
     db = DbWrapper(cfg.CONF.source)
     db.connect()
     tables = db.getSortedTables()
-    for table in tables:
+    try:
```

Comment:
I do not think that is a good idea, as you are hiding the underlining error, like the one from line 255, with a generic 'something is wrong' message, that will require to edit the code to see the real problem when the exception is not an `str`.

What I mean is that this works only for the exception raised in 255, but there are potentially other Exception that are not string based, so printing `e` will not provide information.

I suggest simply to fail here, and capture in the caller with more specific cases.

For the line 255, to break the loop I would suggest to simply use `break` and a `print()` to show the error, instead of the `raise`

Reply:
"For the line 255, to break the loop I would suggest to simply use break and a print() to show the error, instead of the raise"

OK, that makes sense I guess